libs/platform/src/lib/components/list/standard-list-item/standard-list-item.component.html

```diff
@@ -0,0 +1,82 @@
+<!--  list item navigation with Arrow-->
+<ng-container *ngIf="showNavigationArrow">
+    <li #listItem fd-list-item [attr.id]="id" tabindex="-1" [selected]="selected" class="fd-list__item--link"
+        (keydown)="onKeyDown($event)" (keyup)="onKeyUp($event)">
+        <a #link class="fd-list__link fd-list__link--navigation-indicator fd-link fdp-link-truncate__txt"
+            [attr.href]="href" [attr.target]="target">
+            <!-- <ng-container *ngTemplateOutlet="commonTemplate"></ng-container> -->
+            <ng-container *ngTemplateOutlet="hasByLine ? byLineTemplate: commonTemplate"></ng-container>
+        </a>
+    </li>
+</ng-container>
+
+<!-- list item navigation without Arrow-->
+<ng-container *ngIf="!showNavigationArrow && hasNavigation">
+    <li #listItem fd-list-item [attr.id]="id" tabindex="-1" [selected]="selected" class="fd-list__item--link"
+        (keydown)="onKeyDown($event)" (keyup)="onKeyUp($event)">
+        <a #link tabindex="0" class="fd-list__link" [attr.href]="href" [attr.target]="target">
+            <!-- <ng-container *ngTemplateOutlet="commonTemplate"></ng-container> -->
+            <ng-container *ngTemplateOutlet="hasByLine ? byLineTemplate: commonTemplate"></ng-container>
+        </a>
+    </li>
+</ng-container>
```

Comment:
Can we merge these templates by using Angular class binding?

https://angular.io/guide/template-syntax#class-binding

**Example:**
```
[class.fd-list__link--navigation-indicator]="showNavigationArrow"
```

Reply:
Sure, Will update in next commit

Reply:
Done